index.html

```diff
@@ -1,930 +1,127 @@
+					<img id="solIcon" title="Solidity realtime compiler and runtime" src="assets/img/sol.svg" alt="Solidity realtime compiler and runtime">
+				</div>
+				<div id="optionViews" class="settingsView">
+					<div id="txView">
+						<div class="row">
+							<label for="txorigin"><select name="txorigin" id="txorigin"></select> Transaction origin</label>
+						</div>
+						<div class="row hide">
+							<label for="gas"><input type="number" id="gas" value="0"> Gas</label>
+						</div>
+						<div class="row hide">
+							<label for="gasPrice"><input type="number" id="gasPrice" value="0"> Gas Price</label>
+						</div>
+						<div class="row hide">
+							<label for="value"><input type="number" id="value" value="0"> Value</label>
+						</div>
+					</div>
+					<div id="settingsView">
+						<div class="version row"><strong>Solidity version:</strong> <span id="version">(loading)</span></div>
+						<div class="row">Change to: <select id="versionSelector"></select></div>
+						<div class="row">
+							<label for="editorWrap"><input id="editorWrap" type="checkbox">Text Wrap</label>
+							<label for="optimize"><input id="optimize" type="checkbox">Enable Optimization</label>
+						</div>
+					</div>
+					<div id="publishView">
+						<p><button id="gist" title="Publish all files as public gist on github.com"><i class="fa fa-github"></i> Publish Gist</button> Publish all open files to an anonymous github gist.</p>
+						<p>You can also load a gist by adding the following <span class="pre">?gist=GIST_ID</span> to your url, where GIST_ID is the id of the gist to load.</p>
+					</div>
+					<div id="envView">
+						<span id="executionContext">
+							<label for="vm">
+								<input id="vm" type="radio" value="vm" checked name="executionContext">
+								<strong>JavaScript VM</strong>
+								<p>Execution environment does not connect to any node, everything is local and in memory only.</p>
+							</label>
+							<label for="web3">
+								<input id="web3" type="radio" value="web3" name="executionContext">
+								<strong>Web3 Provider</strong>
+                                                                <p>Execution environment connects to node at localhost (or via IPC if available), transactions will be sent to the network and can cause loss of money or worse!<br/>
+                                                                <b>If this page is served via https and you access your node via http, it might not work. In this case, try cloning the repository and serving it via http.</b></p>
+								<label for="web3Endpoint">
+									<strong>Web3 Provider Endpoint</strong>: <input type="text" id="web3Endpoint" value="http://localhost:8545">
+								</label>
+							</label>
+						</span>
+					</div>
+					
+				</div>
+			</div>
+			<div id="output"></div>
+		</div>
+
+		
+
+		<script src="assets/js/app.js"></script>

```

Comment:
It looks like you've extracted the JS from the index.html (been there before :) ) But I cant see `assets/js/app.js` added in this set of commits, where is it?